### DISCUSSION TITLE
install?

### DISCUSSION THREAD

**Discussion by Mauro2022mauro** [MDylan/JW-public-ministry-organizer-app#17] [General]
Sorry my english. I am italian.
How file i must install in my web server?
i do not see a index.html or index.php files.

Thanks

**Comment by MDylan** [comment]
Hi!

Thank you for your question!
Please follow the install [instuctions here](https://github.com/MDylan/JW-public-ministry-organizer-app#installation).

Index.php file is under the "/public" path, so you need to set your document root path to there.
If all set correctly, the first page load will redirect you to installer, when you can set the database, admin user and email settings. 

If you have any questions or problem, ask me again.
